lib/modules/dosomething/dosomething_global/dosomething_global.module

```diff
@@ -6,6 +6,19 @@
 
 include_once 'dosomething_global.features.inc';
 define('GLOBAL_ADMIN_ROLES', 'mexico admin,brazil admin');
+define('GLOBAL_ADMIN_LANGUANGES', serialize(['es-mx' => 'mexico admin', 'pt-br' => 'brazil admin']));
```

Comment:
@angaither Adding that in can be part of refactoring or at least be added in when we do the session/language switching code in conjunction with Fastly.